procedures/auxtel/observation_procedures/DayTime-Checkout.ipynb

```diff
@@ -5,10 +5,10 @@
    "id": "3dbf3c0c",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    if not (corrections.m1 or corrections.hexapod or corrections.atspectrograph):
</pre><p>I don't understand why this statement is needed?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/92/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='48'/>

Reply:
Right, just to confirm that they were disabled in the event. But I guess it's silly since the command `await atcs.rem.ataos.cmd_disableCorrection.set_start(m1=True, hexapod=True, atspectrograph=with_latiss)` already waits for the corrections to be disabled before it completes?

Reply:
Removed.